app/queries/servers/categories.ts

```diff
@@ -19,61 +19,12 @@ export const queryCategoryById = async (database: Database, categoryId: string)
     }
 };
 
-export const queryCategoriesById = async (database: Database, categoryIds: string[]): Promise<CategoryModel[]> => {
-    try {
-        const records = (await database.get<CategoryModel>(CATEGORY).query(Q.where('id', Q.oneOf(categoryIds))).fetch());
-        return records;
-    } catch {
-        return Promise.resolve([] as CategoryModel[]);
-    }
-};
-
-export const queryCategoriesByType = async (database: Database, type: CategoryType): Promise<CategoryModel[]> => {
```

Comment:
not sure why most of this were deleted, I'm going to go with not used anywhere

Reply:
That is the case, correct.